### ISSUE TITLE
Displaying 'areas not studied' for each hazard

### ISSUE THREAD

**Issue by jacobdadams** [agrc/ugs-hazards#20]
Tables 3 & 4 attempted to solve the problem of showing that not finding a hazard doesn't mean it's not there—it could just not yet be mapped. They got complicated and confusing when dealing with AOIs that span different study areas with different sets of studied hazards and were abandoned.

In a future version, perhaps we could have features for every hazard that indicate where it has not been studied and include them as part of the AOI intersection check. Then, on each hazard they would be displayed on the map and included in the description/how-to-use sections.

These new features could be hosted as a separate feature service, using an identical HazardCode to link things up, or be added to the existing feature services as an extension of the HazardUnit (ie, a new 'NS' unit for Not Studied). These both have their pros and cons.

Basically, this idea pivots the studied/not studied logic from being based on study areas (as expressed in the list of hazards studied in each area) to being based on location (as expressed through the new features) and pivots the reporting from tables to additional shapes on the maps.

**Comment by steveoh** [comment]
This is something that UGS would need to develop right? Basically every hazard layer would need to have statewide coverage with no hazard found and not studied attributes on them somewhere.

Is this enhancement something that AGRC can implement or is this more of a wishlist item that we can't fix?

**Comment by jacobdadams** [comment]
Yes, UGS would need to figure out how they want to create and update the non-studied areas and come up with the appropriate text for the description.

This is something that we cannot currently implement with the data we now have. I put it here as a holding place to say this is conversation that needs to be had with UGS. 

I was talking with Nathan about the importance of differentiating between "hazard not present" and "we don't know if it's present or not." Right now the blank areas of our maps (ie, the areas that the AOI doesn't intersect anything) means either of these. If the AOI covers areas both mapped and not mapped, it is not clear what the blank space means, and I know it will be misinterpreted. When it turns out that there is a hazard there, there will be serious trouble, up to lawsuits for someone.

**Comment by gordondouglass** [comment]
We will come up with some text that explains areas that have not been mapped. I think that is the best way to deal e issue.

**Comment by nathankota** [comment]
@gordondouglass How is the text coming along for this? Has it been incorporated into the tables yet?

**Comment by nathankota** [comment]
![image](https://user-images.githubusercontent.com/15659642/70281970-9d8d8280-1779-11ea-9e51-7d7065505b02.png)

**Comment by stdavis** [comment]
So we don't need to make any changes to the app? I've not tested it with a geometry that does not intersect any hazards.

**Comment by nathankota** [comment]
According to #14 the newly-added, statewide, groundshaking layer will always produce a report with at least one hazard. They added some text in the summary to say that just because a hazard doesn't appear on the report, it doesn't mean it's not there - it just may not have been studied.
